packages/jest-preset-mc-app/stylelint-runner/run.js

```diff
@@ -0,0 +1,78 @@
+const { pass, fail } = require('create-jest-runner');
+const stylelint = require('stylelint');
+const postcssImport = require('postcss-import');
+const postcssCustomMediaQueries = require('postcss-custom-media');
+const cssVariables = require('postcss-css-variables');
+const uiKitVariables = require('@commercetools-frontend/ui-kit/materials/custom-properties.json');
+const fs = require('fs');
+const postcss = require('postcss');
+
+const createLinter = testPath => {
+  return options => {
+    const start = new Date();
+
+    return stylelint
+      .lint(options)
+      .then(data => {
+        if (data.errored) {
+          return fail({
+            start,
+            end: new Date(),
+            test: {
+              path: testPath,
+              errorMessage: data.output,
+            },
+          });
+        }
+
+        return pass({
+          start,
+          end: new Date(),
+          test: { path: testPath },
+        });
+      })
+      .catch(err => {
+        throw err;
+      });
+  };
+};
+
+const endsWithAny = (suffixes, string) => {
+  return suffixes.some(function(suffix) {
+    return string.endsWith(suffix);
+  });
+};
+
+module.exports = ({ testPath }) => {
+  const linter = createLinter(testPath);
+  if (endsWithAny(['js', 'jsx', 'ts', 'tsx'], testPath)) {
```

Comment:
Should this come from the runner's config?

Reply:
good call!

Reply:
still not 100% sure of how to pass non jest related configs to a jest runner 🤔